README.md

```diff
@@ -0,0 +1,308 @@
+
+```html
+<div class="row">
+    <div class="form__column form__column--default" bd-form-element>
+        <label class="form__label" for="required">Required</label>
+        <input class="form-element form-element--default" name="required" id="required" required ng-model="vm.formModel.required" />
+    </div>
+</div>
+
+<div class="form__vertical-spacer form__vertical-spacer--medium"></div>
+```
+
+You'll notice we handle vertical spacing using specific spacing elements.
+
+### Individual controls
+
+Individual blocks should not have positioning information or external margin/padding because this makes components difficult to adapt to varying circumstances. 
+For example, if an input element includes a bottom margin, we are forced to create more and more overrides in cases where we want a different size gutter between elements. 
+
+Instead, we strive to write components that take up the full size of their container (100% width) and do not have external margins or padding.
+
+## Consistent order of properties
+
+By keeping a consistent order of properties within a class, it’s easier to quickly read/write an existing class. Basically, it takes away the need to search through the entire class to see if a given property is set.
+
+When writing a class, group different CSS properties together and place them in the following order (credit to Mark Otto @mdo):
+<ul>
+	<li>Positioning</li>
+	<li>Display and box mode</li>
+	<li>Typographic</li>
+	<li>Visual</li>
+</ul>
+
+```css
+.declaration-order {
+    /* Positioning */
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 100;
+
+    /* Display and box model */
+    display: block;
+    float: right;
+    width: 100px;
+    height: 100px;
+    padding: 5px;
+
+    /* Typography */
+    font: normal 13px "Helvetica Neue", sans-serif;
+    line-height: 1.5;
+    color: @offBlack;
+    text-align: center;
+
+    /* Visual */
+    background-color: @lightBlue;
+    border: 1px solid @green;
+    border-radius: 3px;
+}
+
+```
+
+##Specificity
+<ul>
+	<li>Use classes.  No ID's!
+	<li>Elements: Only use them for resetting or noramlizing styles between browsers.
+	<li>If using JS to access a class, prefix it with js-, i.e. js-my-selector and use it only for JS
+	<li>Try to only override a property at most 1 time: </li>
+</ul>
+
+```css
+.my-class {
+	font-size: 12px;
+	text-align: center;
+}
+
+/** this class only overrides a single property. "text-align" remains the same **/
+.my-class--left-text {
+	text-align: left;
+}
+
+```
+
+## Shorthand vs. longhand
+<ul>
+	<li>Define colors in their own less files using upper-case, long-hand Hex format. For example #FFFFFF;</li>
+	<li>Define all sizes in properties that take 4 arguments:  border: 0 0 0 1px vs border-left: 1px;</li>
+	<li>Overriding classes use more specific properties, if possible:  border-left: 1px vs border: 0 0 0 1px;</li>
```

Comment:
I think this reads more consistently with the other two items: "Use more specific properties when overriding classes, if possible"